File: Latihan2_1/Product.java
// class Product adalah parent dari class Hardware
public class Product {
    private String price;
    private String idProduct;
    
    // constructor
    public Product(String price, String idProduct) {
        this.price = price;
        this.idProduct = idProduct;
    }

    // constructor kosong
    public Product() {
    }

    // set product price
    public void set_price(String price) {
        this.price = price;
    }
    // get product price
    public String get_price() {
        return price;
    }

    // set product idProduct
    public void set_idProduct(String idProduct) {
        this.idProduct = idProduct;
    }
    // get product idProduct
    public String get_idProduct() {
        return idProduct;
    }
}

File: Latihan2_2/php/Product.php
<?php

// class Product adalah parent dari class Hardware
class Product
{
    private $price = "";
    private $idProduct = "";

    //constructor
    function __construct(){
    }

    // set memory price
    function set_price($price)
    {
        $this->price = $price;
    }
    // get memory price
    function get_price()
    {
        return $this->price;
    }

    // set memory idProduct
    function set_idProduct($idProduct)
    {
        $this->idProduct = $idProduct;
    }
    // get memory idProduct
    function get_idProduct()
    {
        return $this->idProduct;
    }

    // destructor
    function __destruct(){
    }
}

?>